agent/scripts/fetch-winners.js

```
@@ -0,0 +1,292 @@
+const fs = require('fs');
```

ESLint

[error] 'require' is not defined.

(no-undef)